cro/cro.py

```diff
@@ -131,44 +133,74 @@ def broadcastspawning(self, REEF, REEFpob):
         ESlarvae = np.concatenate([ESlarvae1, ESlarvae2])
         return ESlarvae
 
-    def brooding(self, REEF, REEFpob, type_brooding='op_mutation'):
+    
+    def _larvaemutation(self, larvae, pos, delta=1):
+        """
+        Description:
+            This function modifies the larvae with mutation 
+        Input:
+            - larvae: new individuals to be mutated
+            - pos: selected positions to be mutated
+            - delta: represents an increment or decrement in each mutation (it could be placed as arg in param_grid)
+        Output:
+            - larvae: modified individuals
+        """
+        (nlarvaes, b) = larvae.shape
+        MM = np.zeros([nlarvaes, b], int) # Mutation matrix
+        # int just for disc mode, replace when cont mode takes place
+
+        for key, value in self.param_grid.items():
+            m = value[0]
```

Comment:
Maybe using tuple unpacking makes it easier to read and nicer!
```python
m, M = value
```
Also, `m` and `M` would only store the values of the last iteration right? I still don't fully get the format of `param_grid`

Reply:
Make sense! I think it gave me an error because it isn't a tuple but lets try again!